rental.rb

```diff
@@ -0,0 +1,13 @@
+require './book'
+require './person'
+
+class Rental
+  attr_accessor :date, :book, :person
+
+  def initialize(date, person)
+    @date = date
+    @person = person
```

Comment:
- [ ] nice work. please like the previous reviewer noted, kindly modify the inputs to the initialize method to accept book as a parameter for the many-to-many relationship